notifiers/email.go

```diff
@@ -163,7 +164,13 @@ func (e *emailer) dialSend(email *emailOutgoing) error {
 	}
 
 	m.SetAddressHeader("From", email.From, "Statping")
-	m.SetHeader("To", email.To)
+	//For sending emails to multiple recipients that are comma separated
+	tempSplit := strings.Split(email.To, ",")
+	addresses := make([]string, len(tempSplit))
+	for i := range addresses {
+		addresses[i] = m.FormatAddress(tempSplit[i], "")
+	}
+	m.SetHeader("To", addresses...)
```

Comment:
haven't tested this, I'm thinking of something like
```suggestion
    addresses := strings.Split(email.To, ",")
    m.SetHeader("To", addresses...)
```
(see my long comment from earlier)

Reply:
@jemand771 Thanks for the suggestions. I did get to test your proposed changes as well as the earlier version, both of the cases it does work fine with various email formats, however since your proposed changes are more succinct I agree it makes more sense to use that version. Thank you again for helping to review this.

Reply:
@jemand771 Please review the updated PR.